### PR TITLE
fix: remove leading & trailing slash

### PR DESCRIPTION
**Pull request by lukeed**
Just a precaution against future issues.

We were already taking care against leading slashes – this fixes trailing slashes.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -488,10 +488,7 @@ export default function Vidy(opts) {
 		terms = [];
 		links = [];
 
-		pid = pid || opts.postid;
-		if (pid[0] === '/') {
-			pid = pid.substring(1);
-		}
+		pid = (pid || opts.postid).replace(/^\/|\/$/g, '');
```
Comment (eudisd) [inline src/index.js]: I'm sure this is ok but would remove deep linked routes and query params slahes  ie:

```js
  opts: {
    postid: '/mycustom/blog/2018-10-10?q=1234/234232/234'
  }
```

-->
`postid =  'mycustomblog2018-10-10?q=1234234232234'`

Comment (lukeed) [reply src/index.js]: Nope, it removes the leading (`^\/`) and trailing (`\/$`) only~

Comment (lukeed) [reply src/index.js]: The `g` modifier here just allows it to remove both.

Comment (eudisd) [reply src/index.js]: oh yeah there the ^/$ modifier all good

**Pull request merged**